cryptoconditions/types/ed25519.py

```diff
@@ -57,7 +57,7 @@ def sign(self, message, private_key):
             private_key (string) Ed25519 private key
         """
         sk = private_key
-        vk = VerifyingKey(base58.b58encode(sk.get_verifying_key().to_bytes()))
+        vk = VerifyingKey(base58.b58encode(sk.get_verifying_key().encode(encoding='bytes')))
```

Comment:
Does `get_verifying_key` come from `nacl` inheritance and runs a `nacl.signing.VerifyKey`?
maybe we can override that method to return a `Ed25519.VerifyingKey`?